design-library/src/components/BccGraphic/BccGraphic.stories.ts

```diff
@@ -0,0 +1,34 @@
+import BccGraphic from "./BccGraphic.vue";
+
+import type { Meta, StoryFn } from "@storybook/vue3";
+
+export default {
+  title: "Components/BccGraphic",
+  component: BccGraphic,
+  argTypes: {},
```

Comment:
Storybook unfortunately doesn't automatically recognize union types, which leads to `rounding` looking like this, with no indication what the correct options are:

![image](https://github.com/bcc-code/bcc-design/assets/39056299/e95a65e6-7e00-4f1c-a7dd-0c9831ff1640)

If you add it here, that will be a much nicer experience:

```ts
    rounding: {
      options: ["md", "xl"],
      control: { type: "radio" },
    },
```

Reply:
(see also my comment on renaming these options)